text/0061-promote-ubi-stack.md

```diff
@@ -0,0 +1,56 @@
+# Promote ubi stack to paketo-buildpacks
+
+## Summary
+
+Addition of ubi stacks was discussed and approved in
+[https://github.com/paketo-buildpacks/rfcs/blob/main/text/0056-ubi-based-stacks.md])https://github.com/paketo-buildpacks/rfcs/blob/main/text/0056-ubi-based-stacks.md).
+
+Initially repositories were added to to paketo-community to reflect that they were under early development. 
+
+The ubi stack has been being built and published since August 2023. Except where it implements
+advanced features like multi-arch it uses the common automation, and the team is working to integrate
+the advanced features back into the common versions of the automation flows.  
+
+No major updates are planned to the stack, it already implements multi-arch and the only planned updates
+are to keep it up to date with newer releases of ubi 8.
+
+There are active maintainers for the stack and no significant issues open in the repo.
+
+The stack is ready to be promoted to paketo-buildpacks.
+
+## Motivation
+
+Addition of ubi stacks was approved in
+[https://github.com/paketo-buildpacks/rfcs/blob/main/text/0056-ubi-based-stacks.md])https://github.com/paketo-buildpacks/rfcs/blob/main/text/0056-ubi-based-stacks.md).
+
+Promotion to paketo-buildpacks from paketo-community is the next step towards completing that.
+
+## Detailed Explanation
+
+* Rename https://github.com/paketo-community/ubi-base-stack/ to https://github.com/paketo-community/ubi8-base-stack/
+* Rename all references to paketo-community/ubi-base-stack to paketo-community/ubi8-base-stack across the Paketo projects (go project references and buildpack references) 
+* Rename all references to paketocommunity/ubi-base-stack to paketobuildpacks/ubi8-base-stack across the Paketo projects (Docker containers references)
+
+The rename from ubi to ubi8 is to better reflect that the stack is for ubi8. Just like ubuntu there are different streams of ubi (8, 9, ...) and this is
+a good time to make the change.
+
+## Rationale and Alternatives
+
+The stack is ready to be promoted to paketo-buildpacks there is no obvious reasons not to move to packeto-buildpacks.
+
+Alternative - leave the stack where is in paketo-community and define some additional milestones to be achived before it can be moved.
+
+## Implementation
+
+* Rename https://github.com/paketo-community/ubi-base-stack/ to https://github.com/paketo-community/ubi8-base-stack/
+* Rename all references to paketo-community/ubi-base-stack to paketo-community/ubi8-base-stack across the Paketo projects (go project references and buildpack references)
+* Rename all references to paketocommunity/ubi-base-stack to paketobuildpacks/ubi8-base-stack across the Paketo projects (Docker containers references)
+
+This will take PRs to the two ubi builder repositories, but the main risk is missing updates during the transition. This is manageable and should not
+have a major impact. 
+
+## Prior Art
+
+
+## Unresolved Questions and Bikeshedding
+
```

Comment:
1. How do we plan to communicate this to users?

2. How do we plan to transition users? 

In general, looking for a bit on how can we make this smooth for users and try to ensure users don't get stuck on the Paketo-community images. My concern is someone doesn't get the message and keeps using the old images that stop getting updates.

Reply:
Good questions.

1. I think much of the use of the stack today should be through the ubi builders, those we can switch over easily and those using the builder will end up with the new stack. That should also cover the current integration testing.
2. That leaves people who will have been specifying the stack directly with pack. We have done some blog posts starting with -https://developers.redhat.com/articles/2024/06/18/build-applications-paketo-buildpacks-and-red-hat-ubi-container-images. Those we can update. We can also publish a new blog post as well if we think that will have better change of them being seen
3. Are there communication channels that Paketo has used in the past to get this kind of information out that we can use as well?

Reply:
1. Oh, that's a good point. That should hopefully cover most users.
2. Yes, updating that would be great.
3. Usually we post a blog entry on https://blog.paketo.io/. Are you able to contribute a post? It's a hugo/markdown site so the process is not a lot of work, just make a PR to https://github.com/paketo-buildpacks/blog (more instructions on that README).